src/quantum/subbenchmarks.jsx

```diff
@@ -0,0 +1,121 @@
+
+const Subbenchmarks = ({ match }) => (
+  <div>
+    <PerfherderContainer
+      suite={match.params.suite}
+      platform={match.params.platform}
+    />
+  </div>
+);
+
+Subbenchmarks.propTypes = {
+ match: propTypes.shape({}).isRequired,
+};
+
+class PerfherderContainer extends Component {
+  state = {
+    data: undefined,
+  }
+
+  async componentDidMount() {
+    const { perfherderUrl, data } = await subbenchmarks({
+      platform: this.props.platform,
+      suite: this.props.suite,
+    });
+    // eslint-disable-next-line react/no-did-mount-set-state
+    this.setState({ perfherderUrl, data });
+  }
+
+  async componentDidUpdate(prevProps) {
+    // The component has been called with new props and we
+    // need to update the state or the old state will be used
+    if (this.props.suite !== prevProps.suite) {
+      const { perfherderUrl, data } = await subbenchmarks({
+        platform: this.props.platform,
+        suite: this.props.suite,
+      });
+      // eslint-disable-next-line react/no-did-update-set-state
+      this.setState({ perfherderUrl, data });
+    }
+  }
+
+  render() {
+    const { perfherderUrl, data } = this.state;
+
+    const sortAlphabetically = (a, b) => {
+      if (a.meta.test < b.meta.test) return -1;
+      if (a.meta.test > b.meta.test) return 1;
+      return 0;
+    };
+
+    return (
+      <div className='subbenchmarks align-center'>
+        {!data ?
+          <div>
+            <h3>Loading...</h3>
+          </div> :
+          <div>
+            <div>
+              <div className='header'>
+                <h2 className='wider-letter-spacing'>
+                  {this.props.suite}
+                </h2>
+                <a
+                  className='header-item'
+                  href={perfherderUrl}
+                  target="_blank"
+                  alt='View all subtests on Perfherder'
+                >
+                  <FontAwesomeIcon
+                    icon={faExternalLinkAlt}
+                    style={{ marginLeft: '0.3em' }}
+                  />
+                </a>
+              </div>
+              {Object.values(data).sort(sortAlphabetically).map(el => (
+                <div key={el.meta.test}>
+                  <div className='black-bar'>
+                    <a href={`#${el.meta.test}`}>
+                      <FontAwesomeIcon
+                        id={el.meta.test}
+                        icon={faLink}
+                        style={{ marginRight: '0.3em' }}
+                      />
+                    </a>
+                    <a href={el.meta.url} target="_blank">{el.meta.test}</a>
+                  </div>
+                  <MetricsGraphics
+                    key={el.meta.test}
+                    data={[el.data]}
+                    {...{
+                      x_accessor: 'datetime',
+                      y_accessor: 'value',
+                      min_y_from_data: true,
+                      full_width: true,
```

Comment:
I don't think the chart should be an area chart. Try setting `area: false` here.

I believe that should also take care of the curving issue that jmaher mentioned